ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -40,7 +40,20 @@ if armnn_support_is_available
     nnstreamer_filter_armnn_sources += join_paths(meson.current_source_dir(), s)
   endforeach
 
-  nnstreamer_filter_armnn_deps = armnn_support_deps + [glib_dep, gst_dep, nnstreamer_dep]
+  if cxx.check_header('armnnCaffeParser/ICaffeParser.hpp')
```

Comment:
Just note: ```has_header``` is faster than ```check_header```. https://mesonbuild.com/Reference-manual.html#compiler-object

Reply:
Thanks. Applied.